File: src/main/java/com/jaadla/namlogapi/NamLogApiApplication.java
package com.jaadla.namlogapi;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication
@EnableScheduling
public class NamLogApiApplication {

    public static void main(String[] args) {
        SpringApplication.run(NamLogApiApplication.class, args);
    }

}

File: src/main/java/com/jaadla/namlogapi/NamLogApiApplication.kt
package com.jaadla.namlogapi

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
open class NamLogApiApplication {
}

fun main(args: Array<String>) {
    SpringApplication.run(NamLogApiApplication::class.java, *args)
}
